packages/block-editor/src/hooks/color.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import classnames from 'classnames';
-import { isObject } from 'lodash';
+import { isObject, setWith, clone } from 'lodash';
 
 /**
  * WordPress dependencies
```

Comment:
The `addSaveProps` function needs its logic updated for assigning the `has-link-color` class. At the moment it checks for `style.color.link`.

Reply:
Updated. Right now the server-side hook does not apply this class. I think this class is not useful at all and we should probably remove it in a follow-up PR.